### PR TITLE
[WIP] Additional changes to the new wrapper (next will be the CI)

### PR DESCRIPTION
**Pull request by ProfFan**
This change is [Reviewable](https://reviewable.io/reviews/borglab/gtsam/225)

### CODE REVIEW

File: cmake/GtsamPybindWrap.cmake [hunk 1]
```diff
@@ -69,6 +69,14 @@ function(pybind_wrap
                      APPEND)
 
   pybind11_add_module(${target} ${generated_cpp})
+
+  if(APPLE)
+    # Darker sides of the RTTI system :)
```
Comment (dellaert) [inline cmake/GtsamPybindWrap.cmake]: Please actually explain the issue and the solution, with the links as backup, only. Resist jokes.

Comment (ProfFan) [reply cmake/GtsamPybindWrap.cmake]: Copy that :)

**Comment by ProfFan** [comment]
I will also merge in `develop` to incorporate the new CI for old wrapper.

File: cmake/GtsamPybindWrap.cmake (hunk 1 above)
Comment (ProfFan) [reply cmake/GtsamPybindWrap.cmake]: Done.

**Comment by dellaert** [comment]
I don't understand all the new commits. Did you mix rebase and merge?

**Comment by ProfFan** [comment]
@dellaert No, I just did a `git merge origin/develop`.

**Pull request merged**